src/mplfinance/plotting.py

```diff
@@ -184,6 +184,10 @@ def _valid_plot_kwargs():
         'axtitle'                   : { 'Default'     : None, # Axes Title (subplot title)
                                         'Description' : 'Axes Title (subplot title)',
                                         'Validator'   : lambda value: isinstance(value,(str,dict)) },
+                                        
+        'xlabel'                    : { 'Default'     : 'Date', # x-axis label
+                                        'Description' : 'label for x-axis of main plot',
+                                        'Validator'   : lambda value: isinstance(value,str) },
```

Comment:
A block of code, similar to this, needs to be also added to `_valid_addplot_kwargs()` ... approximately [**here**](https://github.com/matplotlib/mplfinance/blob/master/src/mplfinance/plotting.py#L1300-L1301).